### ISSUE TITLE
Retain Focus on New Track Flyout

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#10381]
### Description

We should allow the User to create multiple tracks without losing focus of the New Track flyout, so that the User doesn't have to navigate through multiple elements to create one more track if that was the User's intention. 

In order to close the New Track flyout, the User can press 'Esc' key to exit the flyout. 

https://github.com/user-attachments/assets/a1ef31f7-638d-426b-bfc1-9fbecb615c84
